### ISSUE TITLE
Selecting single spec from spec list causes all tests to rerun

### ISSUE THREAD

**Issue by edge33** [dfederm/karma-jasmine-html-reporter#46]
Issue description

running ng test in angular project and selecting single spec from spec list causes all tests to rerun
Steps to reproduce the issue

    npm install -g @angular/cli
    ng new test-app
    cd test-app
    ng test
    clic on a single spec

What's the expected result?

    only selected spec gets to run again

What's the actual result?

    all the specs are re run

Additional details / screenshot

Dependencies:

```
"@angular-devkit/build-angular": "~0.1001.4",
"@angular/cli": "~10.1.4",
"@angular/compiler-cli": "~10.1.4",
"@types/node": "^12.11.1",
"@types/jasmine": "~3.5.0",
"@types/jasminewd2": "~2.0.3",
"codelyzer": "^6.0.0",
"jasmine-core": "~3.6.0",
"jasmine-spec-reporter": "~5.0.0",
"karma": "~5.0.0",
"karma-chrome-launcher": "~3.1.0",
"karma-coverage-istanbul-reporter": "~3.0.2",
"karma-jasmine": "~4.0.0",
"karma-jasmine-html-reporter": "^1.5.0",
"protractor": "~7.0.0",
"ts-node": "~8.3.0",
"tslint": "~6.1.0",
```
"typescript": "~4.0.2"

**Comment by jlpstolwijk** [comment]
this should now be fixed in jasmine, see [issue 1906](https://github.com/jasmine/jasmine/issues/1906)., released in jasmine version 3.8.0. Hopefully this repo can soon incoroporate the fixes from there

**Comment by dfederm** [comment]
Pulled in latest from jasmine and published v1.7.0. Seems to have fixed this issue.
